src/server/utils/createCircuitBreaker.js

```diff
@@ -22,12 +22,24 @@ import { registerCircuitBreaker } from '../metrics/circuit-breaker';
 
 const { rootModuleName } = getServerStateConfig();
 let eventLoopDelayThreshold = 250;
+let eventLoopDelayPercentile = 100;
 
 export const setEventLoopDelayThreshold = (n) => {
```

Comment:
maybe set to default when no value given
```suggestion
export const setEventLoopDelayThreshold = (n=eventLoopDelayThreshold) => {
```